INSTALL.md

```diff
@@ -63,38 +62,40 @@ checks locally before uploading your pull request.
 In order to install golangci-lint, you can run:
 
 ```
-$ curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh |
-  sh -s -- -b $(go env GOPATH)/bin v1.15.0
+# binary will be $(go env GOPATH)/bin/golangci-lint
+curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(go env GOPATH)/bin vX.Y.Z
```

Comment:
this is why PATH needs to include ${GOPATH}/bin.

Reply:
Maybe specify vX.Y.Z?

Reply:
Could leave the golangci-lint instructions out since the build process installs a local copy if it is missing.